### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by ashoktvm** [dani-garcia/vaultwarden#3009] [Q&A]
When will a release with new version happens with the latest version of webvault?

**Comment by lukethehawk** [comment]
Hi, I'm not the dev but this is a open source community-mantained project, all the work is done in spare time, so there is not a release date or a schedule

**Comment by nebulade** [comment]
I came here to ask the same question. Since there were new release tags in https://github.com/dani-garcia/bw_web_builds we were wondering if this also means it is compatible with vaultwarden then?

The context is within the Cloudron package, where we package both the backend and frontend together.

Our end-to-end tests worked fine, but we also don't test most of the webvault features to be sure about this.

Any hint would be great so we can deliver timely package updates. Thanks!

**Comment by BlackDex** [comment]
When using the testing tagged images your will have the latest version.
Otherwise you will have to wait until we deem the currently available testing code stable enough for a release.

**Reply by nebulade** [reply]
Thanks for the quick clarification. On Cloudron we build our own images for apps, so then we also just wait for your image tags to be sure.

**Reply by tessus** [reply]
@nebulade The latest git commit (4f0ed06b067540e5a64aa0edaafa346a46cb3ac7) is compatible with the most recent web-vault 2023.4.2
